### PR TITLE
LPD-39278 Not needed

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

File: modules/test/playwright/tests/layout-content-page-editor-web/container.spec.ts [hunk 1]
```diff
@@ -67,3 +72,451 @@ test('Fails to duplicate a container', async ({
...
+
+	await pageEditorPage.selectEditable(duplicatedHeadingId, 'element-text');
+
+	await expect(page.getByLabel('Item', {exact: true})).toHaveValue(
+		'Animal 01 - Dogs and Cats categories'
+	);
+});
+
+test.describe('Container configuration', () => {
+	test('Can change the tag of a container', async ({
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/container.spec.ts:160]: For this one we could use the isolated site right?

File: modules/test/playwright/tests/layout-content-page-editor-web/container.spec.ts [hunk 2]
```diff
@@ -67,3 +72,451 @@ test('Fails to duplicate a container', async ({
...
+		);
+
+		expect(
+			await page
+				.locator('.lfr-layout-structure-item-container')
+				.evaluate((element) => element.tagName)
+		).toBe('ARTICLE');
+	});
+
+	test('Can make a fixed width container', async ({
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/container.spec.ts:215]: Same for this one

File: modules/test/playwright/tests/layout-content-page-editor-web/container.spec.ts [hunk 3]
```diff
@@ -67,3 +72,451 @@ test('Fails to duplicate a container', async ({
...
+		await pageEditorPage.goToConfigurationTab('Styles');
+
+		// Check that the margin left and right fields are disabled
+
+		expect(page.getByLabel('Margin Left')).toBeDisabled();
+
+		expect(page.getByLabel('Margin Right')).toBeDisabled();
+	});
+
+	test('Can configure custom styles', async ({
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/container.spec.ts:268]: This one as well 😇

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts [hunk 4]
```diff
@@ -807,20 +807,28 @@ export class PageEditorPage {
...
 		await this.page.getByRole('tab', {exact: true, name: 'Link'}).click();
 
-		await this.page.locator('select').selectOption({label: 'Mapped URL'});
-
-		await this.waitForChangesSaved();
-
-		await this.page
-			.getByLabel('Field', {exact: true})
-			.selectOption({label: fieldId});
+		this.setLinkConfiguration(linkConfiguration);
```
Comment (sandrodw3) [inline modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts]: Missing `await` right?

Comment (sandrodw3) [reply modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts]: Haven't said anything 🤐

**Review by sandrodw3 — COMMENTED** [review]
Weee amazing 😍 just minor things

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
see: https://github.com/brianchandotcom/liferay-portal/pull/155191

**Pull request closed without merging**